### PR TITLE
Spawner: Notify with hover text on spawning failures

### PR DESCRIPTION
**Pull request by HenryLoenwind**
re #2867

### CODE REVIEW

File: src/main/java/crazypants/enderio/machine/spawner/TilePoweredSpawner.java [hunk 1]
```diff
@@ -61,16 +67,53 @@ public void setSpawnMode(boolean isSpawnMode) {
...
+
   @Override
   protected void taskComplete() {
     super.taskComplete();
     if(isSpawnMode) {
       remainingSpawnTries = Config.poweredSpawnerSpawnCount + Config.poweredSpawnerMaxSpawnTries;
-      for (int i = 0; i < Config.poweredSpawnerSpawnCount && remainingSpawnTries > 0; ++i) {
-        if(!trySpawnEntity()) {
-          break;
+      TRIES: for (int i = 0; i < Config.poweredSpawnerSpawnCount && remainingSpawnTries > 0; ++i) {
```
Comment (tterrag1098) [inline src/main/java/crazypants/enderio/machine/spawner/TilePoweredSpawner.java]: There must be a better way here than using labels and switches...

Comment (HenryLoenwind) [inline src/main/java/crazypants/enderio/machine/spawner/TilePoweredSpawner.java]: I didn't want to redo the original loop logic. And Java uses 'break' for two different things, so...

**Comment by CrazyPants** [comment]
noooo! ;P I had totally forgotten you could still even do that.

File: resources/assets/enderio/lang/en_US.lang [hunk 2]
```diff
@@ -1121,3 +1121,7 @@ enderio.tooltip.meta=Meta:
 enderio.tooltip.uid=UID:
 enderio.tooltip.anvilupgrades=Anvil Upgrades:
 enderio.waila.config.hiddenfacades=Sneaky Facades
+
+enderio.gui.notification.poweredSpawner.bad_entity=BAD SOUL
+enderio.gui.notification.poweredSpawner.area_full=TOO MANY MOBS
+enderio.gui.notification.poweredSpawner.no_location=NO VIABLE SPACE
```
Comment (tterrag1098) [inline resources/assets/enderio/lang/en_US.lang]: All caps is not the convention

File: src/main/java/crazypants/enderio/machine/spawner/TilePoweredSpawner.java (hunk 1 above)
Comment (tterrag1098) [inline src/main/java/crazypants/enderio/machine/spawner/TilePoweredSpawner.java]: I really would prefer a cleaner approach. Using a switch nested in a for loop is ugly to begin with.

**Pull request closed without merging**